x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/querydsl/query/SpatialRelatesQuery.java

```diff
@@ -259,33 +226,19 @@ private static boolean isPointGeometry(XYGeometry[] geometries) {
         private static org.apache.lucene.search.Query shapeShapeQuery(
```

Comment:
The comment above this method is no longer valid, since this code no longer is based on that code, but now calls directly the newer version of that code.

Reply:
removed